src/analysis/souffle/auth_logic_type_interface.dl

```diff
@@ -0,0 +1,49 @@
+//-----------------------------------------------------------------------------
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-----------------------------------------------------------------------------
+
+// This file provides an interface that applies types to the untyped relations 
+// from authorization logic.
+
+
+#ifndef SRC_ANALYSIS_SOUFFLE_AUTHORIZATION_LOGIC_TYPE_INTERFACE_DL_
+#define SRC_ANALYSIS_SOUFFLE_AUTHORIZATION_LOGIC_TYPE_INTERFACE_DL_
+
+#include "dataflow_graph.dl"
+#include "authorization_logic.dl"
+#include "tags.dl"
+
+.decl says_removeTag(speaker: symbol, path: symbol, tag: symbol)
```

Comment:
Thanks! I was planning to create something like this after merging in the ownership PR.

Can you wait for that PR to be merged in?